.eslintrc.json

```diff
@@ -743,6 +743,7 @@
 					"plotly.js",
 					"angular2-grid",
 					"html-query-plan",
+					"html-to-image",
```

Comment:
Did you fix the issue that broke web mode last time?

Reply:
Yes, it's been fixed. The PR has been merged #16845.